build/build_test_only.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This is for test only, please exclude from official merge request

Reply:
Removed